AsyncDisplayKit/ASNetworkImageNode.h

```diff
@@ -96,6 +96,15 @@ NS_ASSUME_NONNULL_BEGIN
 @optional
 
 /**
+ * Notification that the image node started to load
+ *
+ * @param imageNode The sender.
+ *
+ * @discussion Called on a background queue.
+ */
+- (void)imageNodeDidStartLoading:(ASNetworkImageNode *)imageNode;
```

Comment:
Subtle difference but maybe call this imageNodeDidStartFetchingData -- loading is fairly clear but might be viewed as being related to display, and FetchData is the interfaceState terminology to acknowledge local database / disk cache loads.  Please consider alternative names too as I don't love that one!